src/script/components/list/ConversationListCellStatusIcon.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {useMemo} from 'react';
+
+import {Icon} from 'Components/Icon';
+import {useKoSubscribableChildren} from 'Util/ComponentUtil';
+import {t} from 'Util/LocalizerUtil';
+
+import {generateCellState} from '../../conversation/ConversationCellState';
+import {ConversationStatusIcon} from '../../conversation/ConversationStatusIcon';
+import type {Conversation} from '../../entity/Conversation';
+
+export interface ConversationListCellStatusIconProps {
+  conversation: Conversation;
+}
+
+const ConversationListCellStatusIcon = ({conversation}: ConversationListCellStatusIconProps) => {
+  const {
+    participating_user_ets: users,
+    unreadState,
+    mutedState,
+    isRequest,
+  } = useKoSubscribableChildren(conversation, [
+    'isGroup',
+    'is1to1',
+    'selfUser',
+    'participating_user_ets',
+    'display_name',
+    'removed_from_conversation',
+    'availabilityOfUser',
+    'unreadState',
+    'mutedState',
+    'isRequest',
+  ]);
+
+  const cellState = useMemo(() => generateCellState(conversation), [unreadState, mutedState, isRequest]);
+
+  const {isRequest: isUserRequest} = useKoSubscribableChildren(users[0], ['isRequest']);
```

Comment:
How about fixing the `isRequest` of the `Conversation` class so that it does exactly this. 
We could then (and should) remove all traces of `CONVERSATION_TYPE.CONNECT` since it's not reliable anymore

Reply:
https://github.com/wireapp/wire-webapp/pull/15894/commits/80675c242b6346006faceb76575551721973a1cb